example/lib/flap/flap_home_page.dart

```diff
@@ -72,36 +72,14 @@ class _FlapHomePageState extends State<FlapHomePage> {
         Expanded(
           child: AdwScaffold(
             flapController: _flapController,
-            drawer: Drawer(
```

Comment:
I know it's weird but you can't remove this for now :(. If you fire up the example app and reduce the veritcal size of the app, the sidebar will collapse and that drawer will replace it. Without it, no drawer :/.